modules/manage/pages/security/encryption.adoc

```diff
@@ -400,6 +422,8 @@ Redpanda exposes several metrics to help administrators manage their installed c
 
 Configuring alerts on these metrics is a critical tool for managing certificate expiration and avoiding surprise outages. The xref:reference:public-metrics-reference.adoc#tls_metrics[public metrics reference] contains a full list of available TLS metrics. You can refer to the xref:monitoring.adoc[monitor Redpanda] guide for full details on configuring Prometheus to monitor these metrics. This guide also explains how to create a Grafana dashboard for visualizations and alerting.
 
+Alternatively, you can choose to <<configure-tls,specify a certificate revocation list>> to block connections from entities using certifications that have expired or been revoked by a certificate authority (CA). 
```

Comment:
```suggestion
Alternatively, you can choose to <<configure-tls,specify a certificate revocation list>> to reject connections from entities using certifications that have been revoked by a certificate authority (CA). 
```

Reply:
@michael-redpanda, I've slightly reworded this, can you check again?